addons/default/visiosoft/advs-module/resources/views/ad-detail/partials/detail.twig

```diff
@@ -1,5 +1,6 @@
 <div class="slider-text">
-    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 offered-field offered-row">
+    <div class="col-lg-12 col-md-12 col-sm-12 col-xs-12 offered-field offered-row
+        {{ setting_value('visiosoft.module.advs::show_price_to_members_only') ? (auth_check() ? false : 'd-none') : false }}">
```

Comment:
`{{ setting_value('visiosoft.module.advs::show_price_to_members_only') ? (not auth_check() ? 'd-none')  }}`